ansible_base/authentication/social_auth.py

```diff
@@ -152,7 +153,15 @@ def start(self):
         if not self.database_instance.enabled:
             logger.error(f"Authentication attempted with disabled authenticator {self.database_instance.name}")
             return HttpResponseNotFound()
-        return super().start()
+        if self.uses_redirect():
+            auth_url = self.auth_url()
+            log_auth_event(
+                f"Starting SSO redirect to {auth_url} with authenticator '{self.database_instance.name}' (slug: {self.database_instance.slug})",
+                second_logger=logger,
+            )
+            return self.strategy.redirect(auth_url)
+        else:
+            return self.strategy.html(self.auth_html())
```

Comment:
I still think we should call `super().start()` on this one incase the parent library ever changes:
```suggestion
        if self.uses_redirect():
            auth_url = self.auth_url()
            log_auth_event(
                f"Starting SSO redirect to {auth_url} with authenticator '{self.database_instance.name}' (slug: {self.database_instance.slug})",
                second_logger=logger,
            )
        return super().start()
```